sources/packages/backend/apps/workers/src/controllers/supporting-user/_test_/e2e/supporting-user.controller.createIdentifiableSupportingUsers.e2e-spec.ts

```diff
@@ -33,145 +39,187 @@ describe("SupportingUserController(e2e)-createIdentifiableSupportingUsers", () =
     supportingUserController = nestApplication.get(SupportingUserController);
   });
 
-  it("Should create supporting user for the first provided parent and save the associated full name when one parent's information is added to the student application.", async () => {
-    // Arrange
-    const parentFullName = faker.datatype.uuid();
-    const savedApplication = await saveFakeApplication(
-      db.dataSource,
-      undefined,
+  beforeEach(async () => {
+    // Update the date sent for the notifications to current date where the date sent is null.
+    await db.notification.update(
       {
-        applicationData: {
-          workflowName: "some-workflow",
-          parents: [
-            {
-              parentFullName,
-            },
-          ],
-        } as ApplicationData,
+        dateSent: IsNull(),
+        notificationMessage: {
+          id: In([
+            NotificationMessageType.ParentDeclarationRequiredParentCanReportNotification,
+            NotificationMessageType.ParentDeclarationRequiredParentCannotReportNotification,
```

Comment:
Apologies that I missed in the first PR. can we update the enum name to match the notification methods. ?
`ParentInformationRequiredFromStudentNotification` and `ParentInformationRequiredFromParentNotification` ?